src/Components/ListArticles.jsx

```diff
@@ -0,0 +1,17 @@
+import { handlePrevNext } from "../utils"
+
+export default function ListArticles({articles, page, setPage}){
+
+    return (
+        <div>
+            {articles.map(({title, author, topic})=>{
+              return <div key={title + author}>
+                        <h3>{title} -- by {author}</h3>
+                        <p>{topic}</p>
+                    </div>
```

Comment:
Could you render an Article Card component instead? That way you could more easily style them too.